site/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta.adoc

```diff
@@ -126,13 +126,60 @@ Return to the Configure SAML Tab.  This can be found by navigating as follows fr
 
 * Set the value of the [field]#Single sign on URL# to the value of the [field]#Callback URL (ACS)# from the FusionAuth Identity Provider recorded above.
 * Set the value of the [field]#Audience URI (SP Entity ID)# to the value of the [field]#Issuer# recorded above.
-* Optionally set the value of [field]#Default RelayState# if you want to provide a specific redirect URI. If this value is omitted, the first Authorized redirect URI found in the FusionAuth Application OAuth configuration will be used. This option is supported in FusionAuth version greater than or equal to `1.41.0`. In prior versions, append `redirect_uri=` to the value specified [field]#Single sign on URL# to control the final redirect URI.
+* Optionally set the value of [field]#Default RelayState# if you want to provide a specific redirect URI. If this value is omitted, the first Authorized redirect URI found in the FusionAuth Application OAuth configuration will be used. This option is supported in FusionAuth version greater than or equal to `1.41.0`. In prior versions, append `redirect_uri=` to the value specified [field]#Single sign on URL# to control the final redirect URI.  For more information link:/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta#optional-adding-a-specific-redirect-after-saml-login[see section below].
```

Comment:
Suggestion:

>  append a query parameter`redirect_uri=` to the value specified in ...

We may want to note that the value that comes after the equal sign should be URL encoded.